rules/examples/vat-category-Z.xml

```diff
@@ -37,7 +37,7 @@
     </cac:AccountingSupplierParty>
     <cac:AccountingCustomerParty>
         <cac:Party>
-            <cbc:EndpointID schemeID="0184">12345678</cbc:EndpointID>
+            <cbc:EndpointID schemeID="0184">DK12345678</cbc:EndpointID>
```

Comment:
According to https://docs.peppol.eu/edelivery/codelists/v9.4/Peppol%20Code%20Lists%20-%20Participant%20identifier%20schemes%20v9.4.html  , 0184 (DK:DIGST) is exactly 8 digits. It is 0198 (DK:SE) who starts with DK. I guess it is better to omit texts in peppol lookup numbers (e.g. 0192 has not prefix NO), so maybe you keep 0184 here and remove back "DK" prefix?